src/components/views/context_menus/StatusMessageContextMenu.js

```diff
@@ -35,7 +35,7 @@ export default class StatusMessageContextMenu extends React.Component {
         };
     }
 
-    _onClearClick = async (e) => {
+    _onClearClick = async(e) => {
```

Comment:
(This particular linting requirement seems like a bug in our rules, but I am not quite sure where yet...)